azure-pipelines.yml

```diff
@@ -1,7 +1,12 @@
 # Adapted from https://github.com/numba/numba/blob/master/azure-pipelines.yml
 trigger:
-- master
-- 1.2.x
+  branches:
+    include:
+    - master
+    - 1.2.x
+  paths:
+    exclude:
+    - 'doc/*'
```

Comment:
any reason why the other have `doc/**` and this one has `doc/*`?

Reply:
> any reason why the other have `doc/**` and this one has `doc/*`?

Yes, because this is the "azure" CI, and the other files are related to the "github actions" CI.
I just copied the syntax from their examples in the docs, I haven't really tested the differences.